src/App.tsx

```diff
@@ -11,23 +11,27 @@ import { HomePage } from './home-page/HomePage';
 import { theme } from './layout/theme';
 import { ITProfessions } from './ITProfessions';
 
-export const App: React.FC<{}> = () => (
-  <StylesProvider injectFirst>
-    <MuiThemeProvider theme={theme}>
-      <ThemeProvider theme={theme}>
-        <Router>
-          <Switch>
-            <Route exact path="/">
-              <HomePage />
-            </Route>
-            {ITProfessions.map((r) => (
-              <Route exact path={`/${r}`} key={r}>
-                <p>Works {r}</p>
+export const App: React.FC<{}> = () => {
+  return (
+    <StylesProvider injectFirst>
+      <MuiThemeProvider theme={theme}>
+        <ThemeProvider theme={theme}>
+          <Router basename={window.location.pathname || ''}>
```

Comment:
It seems that React Router is not working great with Github pages :/ When we set our basename (either `window.location.pathname` or `/tech-transition`) in `App.tsx`, home page displays fine, but when I try to go to one of our routes it fails: 

![image](https://user-images.githubusercontent.com/8350925/107877623-d7a1fa80-6ecd-11eb-8840-d847b5b4d920.png)

I tried to find a fix today, but no luck so far. Potential solutions are to set  `PUBLIC_URL` for production environment in `.env` or switch to `HashRouter`. For now, I'd merge it anyway to start benefitting from automated `lint` and `test` checks :clap: And add a separate task to fix routing at some point.

Reply:
There is one problem with card links in master, so that can be also a reason why it doesn't work. Card links us to `public_url/frontend`, but route is for `public_url/details/frontend`